database/database.connect.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+const mySecret = process.env['DATABASE_PASSWORD']
+
+const dbConnect = () => {
+  mongoose.connect(`mongodb+srv://joyan11:${mySecret}@cluster0.4n12u.mongodb.net/ecommerce?retryWrites=true&w=majority`, { useNewUrlParser: true, useUnifiedTopology: true, useFindAndModify: false }).then(() => console.log("MongoDB connected")).catch((err) => console.error(err))
+}
+
+
+module.exports = { dbConnect };
```

Comment:
A nice way of making `dbConnect` function reusable can be like passing `MONGODB_URI` as a parameter to the function. In that way, you can use this function in all those projects which uses `MongoDB`.